ncm-filecopy/src/main/perl/filecopy.pm

```diff
@@ -149,7 +149,12 @@ sub Configure
                                     stderr => "stdout");
         $cmd->execute();
         if ( $? ) {
-            $self->error("Command failed. Command output: $cmd_output\n");
+            my $message = "Command failed. Command output: $cmd_output\n";
```

Comment:
more perlish (and it's `warn`, not `warning`; if this annoys you, open an issue in CAF)
```perl
my $method = $confighash->{ignore_restart_failure} ? 'warn' ? 'error'
$self->$method("Command failed. Command output: $cmd_output\n");
```